presqt/api_v1/utilities/multiprocess/watchdog.py

```diff
@@ -0,0 +1,41 @@
+from time import sleep
+
+from presqt.api_v1.utilities import write_file
+from presqt.api_v1.utilities.io.read_file import read_file
+
+
+def process_watchdog(process_function, process_info_path, process_time, process_state):
+    """
+    Watchdog for file transfer processes spawned off using Multiprocessing. It will monitor if the
```

Comment:
Channeling Jared: You probably shouldn't use Watchdog when define what `process_watchdog` is. It's a circular definition.